frontend/src/renderer/components/Process/Dropbox.svelte

```diff
@@ -9,27 +9,30 @@
     export let items = [];
     export let type;
     export let singleItem = true;
-    export let showError = false;
+    export let minimum = true;
 
     function handleSort(e) {
         items = e.detail.items;
     }
-
 </script>
-<main>
-    <div class="sectionStyle {showError && singleItem && isEmpty(items) ? 'errorStyle' : ''}">
-        <section use:dndzone={{items, flipDurationMs, type, dropFromOthersDisabled: singleItem && items.length > 0}}
-            on:consider={handleSort}
-            on:finalize={handleSort}
-        >
-            {#each items as item (item?.id)}
-                <card animate:flip={{ duration: flipDurationMs }} class="{singleItem ? 'verified' : ''}">
+
+<div class="sectionStyle">
+    <section use:dndzone={{items, flipDurationMs, type, dropFromOthersDisabled: singleItem && items.length > 0}}
+        on:consider={handleSort}
+        on:finalize={handleSort}
+    >
+        {#each items as item (item?.id)}
+            <card animate:flip={{ duration: flipDurationMs }} class="{singleItem ? 'verified' : ''}">
```

Comment:
is this verified thing what makes the image draggable green?

Reply:
I believe so